data/autoyast_sle12sp2/post_script.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
```

Comment:
Sounds good. I got this script from aytests repo, so we should back merge it. I'll mention this in the progress ticket https://progress.opensuse.org/issues/20064